src/Windows.Core/DEVMODE.cs

```diff
@@ -0,0 +1,1326 @@
+        /// </summary>
+        DMDO_DEFAULT = 0,
+
+        /// <summary>
+        /// The display device orientation is 90 degrees (measured clockwise) from that of <see cref="DMDO_DEFAULT"/>.
+        /// </summary>
+        DMDO_90 = 1,
+
+        /// <summary>
+        /// The display device orientation is 180 degrees (measured clockwise) from that of <see cref="DMDO_DEFAULT"/>.
+        /// </summary>
+        DMDO_180 = 2,
+
+        /// <summary>
+        /// The display device orientation is 270 degrees (measured clockwise) from that of <see cref="DMDO_DEFAULT"/>.
+        /// </summary>
+        DMDO_270 = 3,
+    }
+
+    public enum DEVMODE_DisplayFixedOutputOptions : uint
+    {
+        DMDFO_DEFAULT = 0,
+
+        /// <summary>
+        /// The display device presents a lower-resolution mode image by stretching it to fill the larger screen space.
+        /// </summary>
+        DMDFO_STRETCH = 1,
+
+        /// <summary>
+        /// The display device presents a lower resolution mode image by centering it in the larger screen space.
+        /// </summary>
+        DMDFO_CENTER = 2,
+    }
+
+    public enum DEVMODE_PrintColorOptions : short
+    {
+        DMCOLOR_MONOCHROME = 1,
+        DMCOLOR_COLOR = 2,
+    }
+
+    public enum DEVMODE_PrintDuplexOptions : short
+    {
+        /// <summary>
+        /// Print single-sided.
+        /// </summary>
+        DMDUP_SIMPLEX = 1,
+
+        /// <summary>
+        /// Print double-sided, using long edge binding.
+        /// </summary>
+        DMDUP_VERTICAL = 2,
+
+        /// <summary>
+        /// Print double-sided, using short edge binding.
+        /// </summary>
+        DMDUP_HORIZONTAL = 3,
+    }
+
+    public enum DEVMODE_TrueTypeOptions : short
+    {
+        /// <summary>
+        /// Print TT fonts as graphics.
+        /// </summary>
+        DMTT_BITMAP = 1,
+
+        /// <summary>
+        /// Download TT fonts as soft fonts.
+        /// </summary>
+        DMTT_DOWNLOAD = 2,
+
+        /// <summary>
+        /// Substitute device fonts for TT fonts.
+        /// </summary>
+        DMTT_SUBDEV = 3,
+
+        // #if (WINVER >= 0x0400)
+
+        /// <summary>
+        /// Download TT fonts as outline soft fonts.
+        /// </summary>
+        DMTT_DOWNLOAD_OUTLINE = 4,
+    }
+
+    public enum DEVMODE_CollationSelectionOptions : short
+    {
+        /// <summary>
+        /// Do not collate when printing multiple copies.
+        /// </summary>
+        DMCOLLATE_FALSE = 0,
+
+        /// <summary>
+        /// Collate when printing multiple copies.
+        /// </summary>
+        DMCOLLATE_TRUE = 1,
+    }
+
+    /// <summary>
+    /// Specifies the device's display mode in the <see cref="DEVMODE.dmDisplayFlags"/> field.
+    /// This member can be a combination of the following values.
+    /// </summary>
+    [Flags]
+    public enum DEVMODE_DisplayOptions : uint
```

Comment:
With Flags enums, sometimes there is value in having a `None=0x0` value. It depends on how the corresponding Win32 function uses these flags. If the receiving function sometimes uses 0 to indicate no-flags, then introducing a `None` here (and in other Flags enums) could help the consumers of these API's.

Reply:
Done.